src/stories/1-Button.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Let's remove this since it's not useful for the app.

Reply:
Done!